conceptual.md

```diff
@@ -3,22 +3,37 @@
 Answer the following questions below:
 
 - What are some ways of managing asynchronous code in JavaScript?
+  * Callback functions
+  * Promises
+  * Async/Await
 
 - What is a Promise?
+  * A one-time guarantee of a future value
```

Comment:
Equally important is that it executes code concurrently (i.e. doesn't block the main thread).